src/test/java/de/tum/in/www1/artemis/programmingexercise/ProgrammingExerciseIntegrationTestService.java

```diff
@@ -270,6 +270,34 @@ void testProgrammingExerciseIsReleased_forbidden() throws Exception {
         request.get("/api/programming-exercises/" + programmingExercise.getId() + "/test-case-state", HttpStatus.FORBIDDEN, Boolean.class);
     }
 
+    void testExportSubmissionsByParticipationIds_excludePracticeSubmissions() throws Exception {
```

Comment:
I think there should be a second test for including all submissions when the `exclude...` setting is `false`

Reply:
True. I thought that existing tests cover it enough but you are right that there is actually no test which includes a `testRun==true` practice session besides my first test. I will add one.